SabreTools.Models/SecuROM/DFAFile.cs

```diff
@@ -0,0 +1,114 @@
+namespace SabreTools.Models.SecuROM
+{
+    /// <summary>
+    /// Most DFA-protected files seem to also have some additional encryption (it looks like SecuROM DFE, but that
+    /// could be an incorrect guess); only early RC-encrypted executables seem to have anything parsable beyond the
+    /// SDFA header. When I mention "observable cases" this is what I mean.
+    /// </summary>
+    public class DFAFile
```

Comment:
Overarching comment: Please be consistent with capitalization.

Reply:
Secondary overarching comment: You can put the "name given by subheader" comments in `<remarks>` tags